src/charm_state.py

```diff
@@ -21,6 +22,20 @@
 from charm_types import WebserverConfig
 from exceptions import CharmConfigInvalidError
 
+KNOWN_CHARM_CONFIG = (
```

Comment:
It doesn't have to be, but the list of config variables is not sorted, perhaps it could add to readability.

Reply:
Sorted, and I sorted the configurations in `config.yaml` too.